cmd/dexctl/command_config.go

```diff
@@ -81,10 +82,36 @@ func runGetConnectorConfigs(cmd *cobra.Command, args []string) int {
 	fmt.Printf("Found %d connector config(s)\n", len(cfgs))
 
 	for _, cfg := range cfgs {
-		fmt.Println()
-		fmt.Printf("ID:   %v\n", cfg.ConnectorID())
-		fmt.Printf("Type: %v\n", cfg.ConnectorType())
+		switch typedCFG := cfg.(type) {
+		case map[string]interface{}:
+
+			id := typedCFG["id"].(string)
```

Comment:
The following program panics.

```
package main

import "fmt"

func main() {
	m := map[string]interface{}{"id": 4}
	id := m["id"].(string)
	fmt.Println(id)
}
```

Please ensure that `"id"` is actually a string (`id, ok := typedCFG["id"]`)